### PR TITLE
fix(wrw): remove gas fields from EVM CCR FORM and bump sdk

### PR DESCRIPTION
**Pull request by venkateshv1266**
[COIN-1201](https://bitgoinc.atlassian.net/browse/COIN-1201)

Remove gasLimit, gasPrice, maxFeePerGas and maxPriorityFeePerGas from EVM CCR FORM. 

This requires SDK change of the PR https://github.com/BitGo/BitGoJS/pull/4905

[COIN-1201]: https://bitgoinc.atlassian.net/browse/COIN-1201?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/containers/EvmCrossChainRecoveryWallet/EvmCrossChainRecoveryBaseForm.tsx [hunk 1]
```diff
@@ -132,8 +124,8 @@ export function EvmCrossChainRecoveryBaseForm({
       {!isCustodyWallet && (
         <div className="tw-mb-4">
           <FormikTextfield
-            HelperText="Gas limit for the Polygon transaction. The value should be between 30,000 and 20,000,000. The default is 500,000 unit of gas."
-            Label="Gas Limit"
+            HelperText="Gas limit for the Polygon transaction. The value should be between 30,000 and 20,000,000."
```
Comment (mullapudipruthvik) [inline src/containers/EvmCrossChainRecoveryWallet/EvmCrossChainRecoveryBaseForm.tsx:127]:
```suggestion
            HelperText="Gas limit for the transaction. The value should be between 30,000 and 20,000,000."
```

File: src/containers/EvmCrossChainRecoveryWallet/EvmCrossChainRecoveryBaseForm.tsx [hunk 2]
```diff
@@ -144,17 +136,17 @@ export function EvmCrossChainRecoveryBaseForm({
         <>
           <div className="tw-mb-4">
             <FormikTextfield
-              HelperText="Max fee per gas for the Polygon transaction. The default is 20 Gwei."
-              Label="Max Fee Per Gas (Gwei)"
+              HelperText="Max fee per gas for the Polygon transaction."
```
Comment (mullapudipruthvik) [inline src/containers/EvmCrossChainRecoveryWallet/EvmCrossChainRecoveryBaseForm.tsx:139]:
```suggestion
              HelperText="Max fee per gas for the transaction."
```

File: src/containers/EvmCrossChainRecoveryWallet/EvmCrossChainRecoveryBaseForm.tsx [hunk 3]
```diff
@@ -144,17 +136,17 @@ export function EvmCrossChainRecoveryBaseForm({
...
               name="maxFeePerGas"
               Width="fill"
             />
           </div>
 
           <div className="tw-mb-4">
             <FormikTextfield
-              HelperText='"Tip" to the Polygon miner. This is by default 10 Gwei.'
-              Label="Max Priority Fee Per Gas (Gwei)"
+              HelperText='"Tip" to the Polygon miner.'
```
Comment (mullapudipruthvik) [inline src/containers/EvmCrossChainRecoveryWallet/EvmCrossChainRecoveryBaseForm.tsx:148]:
```suggestion
              HelperText='"Tip" to the miner.'
```

File: src/containers/EvmCrossChainRecoveryWallet/EvmCrossChainRecoveryBaseForm.tsx [hunk 4]
```diff
@@ -165,8 +157,8 @@ export function EvmCrossChainRecoveryBaseForm({
       {!isCustodyWallet && isBscChain(wrongChain) && (
         <div className="tw-mb-4">
           <FormikTextfield
-            HelperText="Provide the Gas Price for BSC. This is by default 20 Gwei"
-            Label="Gas Price (Gwei)"
+            HelperText="Provide the Gas Price for BSC. By default it takes the gasPrice from Explorer"
```
Comment (mullapudipruthvik) [inline src/containers/EvmCrossChainRecoveryWallet/EvmCrossChainRecoveryBaseForm.tsx:160]:
```suggestion
            HelperText="Provide the Gas Price for the transaction."
```

File: src/containers/EvmCrossChainRecoveryWallet/EvmCrossChainRecoveryBaseForm.tsx (hunk 1 above)
Comment (venkateshv1266) [reply src/containers/EvmCrossChainRecoveryWallet/EvmCrossChainRecoveryBaseForm.tsx]: Done

File: src/containers/EvmCrossChainRecoveryWallet/EvmCrossChainRecoveryBaseForm.tsx (hunk 2 above)
Comment (venkateshv1266) [reply src/containers/EvmCrossChainRecoveryWallet/EvmCrossChainRecoveryBaseForm.tsx]: Done

File: src/containers/EvmCrossChainRecoveryWallet/EvmCrossChainRecoveryBaseForm.tsx (hunk 3 above)
Comment (venkateshv1266) [reply src/containers/EvmCrossChainRecoveryWallet/EvmCrossChainRecoveryBaseForm.tsx]: Done

File: src/containers/EvmCrossChainRecoveryWallet/EvmCrossChainRecoveryBaseForm.tsx (hunk 4 above)
Comment (venkateshv1266) [reply src/containers/EvmCrossChainRecoveryWallet/EvmCrossChainRecoveryBaseForm.tsx]: Done

**Review by mullapudipruthvik — APPROVED** [review]

**Pull request merged**